cms/djangoapps/contentstore/views/course.py

```diff
@@ -536,41 +535,6 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
-def _deprecated_blocks_info(course_module, deprecated_block_types):
-    """
-    Returns deprecation information about `deprecated_block_types`
-
-    Arguments:
-        course_module (CourseDescriptor): course object
-        deprecated_block_types (list): list of deprecated blocks types
-
-    Returns:
-        Dict with following keys:
-        block_types (list): list containing types of all deprecated blocks
-        block_types_enabled (bool): True if any or all `deprecated_blocks` present in Advanced Module List else False
-        blocks (list): List of `deprecated_block_types` component names and their parent's url
-        advance_settings_url (str): URL to advance settings page
-    """
-    data = {
-        'block_types': deprecated_block_types,
-        'block_types_enabled': any(
-            block_type in course_module.advanced_modules for block_type in deprecated_block_types
-        ),
-        'blocks': [],
-        'advance_settings_url': reverse_course_url('advanced_settings_handler', course_module.id)
-    }
-
-    try:
-        structure_data = api.course_structure(course_module.id, block_types=deprecated_block_types)
```

Comment:
Assuming this code needs to stay, this line needs to be updated to use the new course-block api instead of the course_structure api.